package.json

```diff
@@ -152,9 +152,9 @@
     "css-loader": "^0.28.11",
     "del": "^3.0.0",
     "devtron": "^1.4.0",
-    "electron": "3.0.8",
-    "electron-builder": "^20.19.2",
-    "electron-debug": "^1.5.0",
+    "electron": "^3.0.9",
```

Comment:
I'm sorry, I forgot to mention. Please wait to upgrade electron.

I want to specify electron version. Because sometimes it contains breaking changes, so I would like to control upgrading after I read the change logs.

Reply:
I've reverted that particular bit. I'll keep it in mind!